pkgs/development/python-modules/spotify-cli-linux/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonApplication, fetchPypi, isPy3k, lyricwikia, dbus-python }:
+buildPythonApplication rec {
+  pname = "spotify-cli-linux";
```

Comment:
Why you need this file? It duplicates `pkgs/applications/audio/spotify-cli-linux/default.nix`.

Reply:
Happened accidentally. Removed in the most recent commit.